Abort is a regular command (#15669)
coq/coq: doc/changelog/07-vernac-commands-and-options/15669-abort-time.rst

- **Removed:**

>   :cmd:`Abort` does not take an identifier as argument (it was ignored since 8.5)

Reviewer: ```suggestion
  :cmd:`Abort` no longer takes an identifier as argument (it was ignored since 8.5)
```

>   :cmd:`Abort` no longer takes an identifier as argument (it was ignored since 8.5)

Reviewer 2: ```suggestion
  :cmd:`Abort` no longer takes an :n:`@ident` as argument (it has been ignored since 8.5)
```

>   :cmd:`Abort` no longer takes an :n:`@ident` as argument (it has been ignored since 8.5)

Reviewer 2: ```suggestion
  :cmd:`Abort` no longer takes an :n:`@ident` as an argument (it has been ignored since 8.5)
```
